Makefile

```diff
@@ -26,23 +25,17 @@ check_tools:
 get_tools:
 	@echo "--> Installing tools"
 	go get -u -v $(GOTOOLS)
-	@gometalinter.v2 --install
 
 update_tools:
 	@echo "--> Updating tools"
 	@go get -u $(GOTOOLS)
 
-get_vendor_deps:
-	@rm -rf vendor/
-	@echo "--> Running dep ensure"
-	@dep ensure
-
 
 ########################################
 ### Testing
 
 test:
-	go test $(shell go list ./... | grep -v vendor)
+	go test -v $(shell go list ./... | grep -v vendor)
```

Comment:
why add verbose flag?

Reply:
I wasn't seeing what exactly was going on with the failing tests. Can revert though.